islandora_image.info.yml

```diff
@@ -0,0 +1,5 @@
+name: islandora_image
+type: module
+description: Islandora Image
+core: 8.x
+package: islandora
```

Comment:
@kimpham54 I have one more, could you add a dependency for islandora here. Looks like this https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/islandora.info.yml#L8-L17

But you only need islandora, so
```
dependencies:
  - islandora
```